mlibrary/catalog-browse
# [LIBSEARCH-436] Implement design for author browse

# Overview
Data now exists for Author Browse. This pull request updates the UI to the latest [Figma designs](https://www.figma.com/file/ymqm7jMhh1xFIONkNZZrcB/Browse-Search-Application-Designs?node-id=722%3A17393). The missing pieces of information are the number of records and the `About this author` links for every cross reference.

This pull request resolves closes [LIBSEARCH-436](https://mlit.atlassian.net/browse/LIBSEARCH-436).

## Beta Testing
The `m-callout` messaging has been updated to change based on which option is being browsed.

## External Link
An `external_link` component has been added since there are many links that open in a new tab. This component allows developers to add it in a consistent and accessible way. The component allows three locals: `url`, `text`, and `classes`.

## Material Icons
Design System will be retiring `m-icon`. In an effort to remove its use in the app, Material Icons has been added. For now it is only used in the `external_link` component.

## Testing
- Run the tests to make sure they pass (`docker-compose run --rm web bundle exec rspec`).
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Search for [an author](http://localhost:4567/author?query=Twain%2C+Mark)
  - Do you see the updated layout?
  - Does the beta message update?
